addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -33,7 +33,7 @@ $o-we-border-color: $o-we-bg-light !default;
 
 $o-we-ace-color: #2F3129 !default;
 
-$o-we-handles-offset-to-hide: 10000px !default;
+$o-we-handles-offset-to-hide: 1.7895e7px !default; // Max value before handle break on firefox
```

Comment:
Sorry to be annoying but maybe I was not clear: it's better with a comment indicating that this is the max value but... do you know why or did you find any doc about it (if yes, this is what this comment should be about)? 10000px has been working for 8 years, we have to be careful. If you found that new value by trying randomly, I would not use it, and round it down to feel less weird. Then the comment can only be about why it needs to be high. Trying to choose the maximum value by chance will only have the bad effect of risking that max value to be changed by Firefox in 3 months and have *all websites broken* instead of one sad customer in 8 years.

So this:
```scss
$o-we-handles-offset-to-hide: 1.7895e7px !default; // Max value before handle break on firefox
```
- When reading this, the reader ask itself: "where does that value comes from?", "why does it need to be that high?"
- If random, risk of Firefox changing that value and all databases will have a bug

So if there is no apparent precise value I would do:
```scss
// This needs to be a high enough value so that it is greater than the maximum
// size the page of a customer could be (10000px was proven to be too small).
$o-we-handles-offset-to-hide: 100000px !default;
```
- Lesser risk of that value not working in any browser
- The comment explains why it is needed to be that high

Reply:
Ok, I will change the value to 100000px. For the commit message, what should I change? I think the new message is following the guideline. Or is it because I didn't edit the PR message?

Reply:
> Max 72 characters per line